app/models/Subscriber.scala

```diff
@@ -346,6 +346,25 @@ case class Subscriber(id: Int,  // DB key
       ).on('id -> id, 'userId -> userId).executeUpdate
     }
   }
+
+  def delete = {
```

Comment:
This was largely created as a convenience in testing at this point, but ultimately we may want to expose this functionality in the UI as well. When/if we do that, we may want to consider whether the foreign_key should be set to cascade the delete or whether we prefer to handle the cleanup in our application like this.